.github/workflows/nucliadb.yml

```diff
@@ -78,6 +79,7 @@ jobs:
         uses: nick-fields/retry@v2
         env:
           TESTING_MAINDB_DRIVERS: ${{ matrix.maindb_driver }}
+          RELEASE_CHANNEL: ${{ matrix.release_channel }}
```

Comment:
Can you make it explicit that is for testing? 
like: `TESTING_NUCLIADB_CHANNEL` or something alike